test/cmd/dns.sh

```diff
@@ -1,48 +0,0 @@
-#!/bin/bash
```

Comment:
> cc @ironcladlou for approval on this deletion

More like notification.  The kube-apiserver doesn't include a DNS server.

Reply:
These tests look like a less comprehensive version of the origin DNS e2e tests which are themselves now probably superseded by upstream e2e tests, so I'm not sure there's any good reason to keep these. /cc @smarterclayton since he seems to know the most about the existing DNS tests.